### PR TITLE
[New] Using the export command 

### PR DESCRIPTION
**Pull request by Rajakavitha1**
provided examples to use export command and the available options

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 7e8a1fb3da5d307ecb795b8108dc16f83ff5c1a0

https://deploy-preview-3387--nostalgic-ptolemy-b01ab8.netlify.app

File: docs/quick-answers/linux/how-to-use-export/index.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+modified: 2020-06-08
+modified_by:
+  name: Linode
+published: 2020-06-08
+title: How to Use Export
+h1_title: Using the export Command
+---
+Export command is used to pass variables to other processes.
+
+When you open a new terminal session you set environment variables for the process. At any time if you change any of the variable values, the terminal does not pick that change. The export command updates the current terminal session about the change you made to the exported variable. You don’t have to open a new terminal session to use the value of the variable that you changed.
```
Comment (andystevensname) [inline docs/quick-answers/linux/how-to-use-export/index.md:18]: ...the terminal does not pick _up_ that change...

File: docs/quick-answers/linux/how-to-use-export/index.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+aliases: ['quick-answers/how-to-use-export/']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: 2020-06-08
+modified_by:
+  name: Linode
+published: 2020-06-08
+title: How to Use Export
+h1_title: Using the export Command
+---
+Export command is used to pass variables to other processes.
```
Comment (andystevensname) [inline docs/quick-answers/linux/how-to-use-export/index.md:16]: _The_ `export` command...

File: docs/quick-answers/linux/how-to-use-export/index.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    declare -x USER="linode"
+
+
+### Export names as functions
+
+To export names as functions use the following command syntax:
+
+    export -f < function_name >
+
+   If you do not use the option `-f` the names is treated as variables.
```
Comment (andystevensname) [inline docs/quick-answers/linux/how-to-use-export/index.md:84]: ...the names  _are_ treated as variables...

File: docs/quick-answers/linux/how-to-use-export/index.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
+---
+author:
+  name: Rajakavitha Kodhandapani
+  email: docs@linode.com
+description: 'This Quick Answer guide will explain how to use export.'
```
Comment (andystevensname) [inline docs/quick-answers/linux/how-to-use-export/index.md:5]: Maybe expand this? "This Quick Answer guide will explain how to use the export command to view terminal variables, how to  assign variable values, and how to pass functions to other processes."

Also, we need an og_description.

**Review by andystevensname — APPROVED** [review]

**Review by wildmanonline — APPROVED** [review]

**Comment by andystevensname** [comment]
@Rajakavitha1 can you update the dates on this guide? And then we will merge it in.

**Comment by wildmanonline** [comment]
@Rajakavitha1 I updated this PR with my tech/copy edit. That said, I just found a similar guide that has already been published: [Setting and Using Linux Environment Variables](https://www.linode.com/docs/guides/how-to-set-linux-environment-variables/). As such, I'm going to close out this PR. Feel free to reopen if you think otherwise.

**Pull request closed without merging**